webui/react/src/services/utils.ts

```diff
@@ -97,12 +97,12 @@ export function generateApi<Input, Output>(api: HttpApi<Input, Output>) {
 }
 
 export function generateDetApi<Input, DetOutput, Output>(api: DetApi<Input, DetOutput, Output>) {
-  return async function(params: Input & ApiCommonParams): Promise<Output> {
+  return async function(params: Input & OptionParams): Promise<Output> {
     try {
       const response = api.stubbedResponse ? api.stubbedResponse : await api.request(params);
       return api.postProcess(response);
     } catch (e) {
-      processApiError(api.name, e);
+      if (e.name !== 'AbortError') processApiError(api.name, e);
```

Comment:
suggestion: add a helper, much like `isNotFound` since we'd be using this check in other places in our codebase as well

Reply:
great idea